src/utils/redux/middleware/history.js

```diff
@@ -13,12 +13,8 @@ export const history = (log: Object[] = []) => ({
   dispatch,
   getState
 }: ThunkArgs) => {
-  if (isDevelopment()) {
-    console.warn(
-      "Using history middleware stores all actions in state for " +
-        "testing and devtools is not currently running in test " +
-        "mode. Be sure this is intentional."
-    );
+  if (!isDevelopment()) {
+    return;
```

Comment:
lets flip it so we know we can't log history in production

Reply:
👏